File: CodingDojo/bomberman-servers/java/src/main/java/com/Direction.java
package com;

public class Direction {

    public static final String UP = "UP";
    public static final String DOWN = "DOWN";
    public static final String LEFT = "LEFT";
    public static final String RIGHT = "RIGHT";
    public static final String BOMB = "ACT";
    private Board board;

    public Direction(Board board) {
        this.board = board;
    }

    public String get() {
        return BOMB;
    }
}

File: CodingDojo/games/battlecity/src/main/javascript/games/battlecity/direction.js
var BattlecityDirection = module.exports = function() {

    var Direction = require('./../../direction.js');

    // move numbers
    Direction.LEFT = Direction._init(1, -1, 0, 'left');
    Direction.RIGHT = Direction._init(2, 1, 0, 'right');
    Direction.UP = Direction._init(3, 0, 1, 'up');
    Direction.DOWN = Direction._init(4, 0, -1, 'down');

    // fire
    Direction.ACT = Direction._init(5, 0, 0, 'act');

    return Direction;
}